pyproject.toml

```diff
@@ -21,7 +25,6 @@ addopts = [
   "--tb=native",
   "-vv",
 ]
-testpaths = ["**/"]
```

Comment:
For some reason this doesn't find `test_read_file.py` but the pytest's default recursive search finds the tests